src/w3iProxy/pushProviders/internalPushProvider.ts

```diff
@@ -124,9 +126,41 @@ export default class InternalPushProvider implements W3iPushProvider {
     if (!this.pushClient) {
       throw new Error(this.formatClientRelatedError('subscribe'))
     }
-
     console.log('InternalPushProvider > PushClient.subscribe > params', params)
 
+    try {
+      Notification.requestPermission()
```

Comment:
I just realised this is a promise, so this isn’t handling the actual result of the perms request, can we fix please: see eg:
https://developer.mozilla.org/en-US/docs/Web/API/Notification/requestPermission_static#examples